gosling/examples/heatmap.py

```diff
@@ -9,10 +9,12 @@
 data = gos.matrix("https://server.gosling-lang.org/api/v1/tileset_info/?d=hffc6-microc-hg38")
 # data = gos.matrix('/path/to/dataset.cool') # local dataset
     
-track = gos.Track(data).mark_rect().encode(
-  x="position1:G",
-  y="position2:G",
-  color=gos.Color("value:Q", range="warm"),
+track = gos.Track(data).mark_bar().encode(
+  x="xs:G",
+  xe="xe:G",
+  y="ys:G",
```

Comment:
```suggestion
  x=gos.X("xs:G", axis="bottom"),
  xe="xe:G",
  y=gos.Y("ys:G", axis="left"),
```

Reply:
We can set `axis` to display the y-axis. 

I think Gosling.js should display the y-axis by default in the future.